eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/parameters/InputParameters.java

```diff
@@ -47,6 +49,7 @@ public JobParameters toJobParameters() {
                 .addString("assemblyReportUrl", assemblyReportUrl)
                 .addLong("buildNumber", buildNumber)
                 .addString("outputVcf", outputVcf)
+                .addString("outputVcfMerged", outputVcfMerged)
```

Comment:
Or we could make it a bit easier to use by deriving the merged and deprecated file names from the `outputVcf` parameter.